package/utils/configPath.js

```
@@ -0,0 +1,5 @@
+const path_1 = require("path")
+const configPath =
+  process.env.SHAKAPACKER_CONFIG ||
+  (0, path_1.resolve)("config", "shakapacker.yml")
```

ESLint

[error] Identifier 'path_1' is not in camel case.

(camelcase)